README.md

```diff
@@ -11,9 +11,24 @@ This is where all posting magic happens.
 ## [UI](/ui)
 React code that handles UI of the application displayed in the desktop application.
 
+## Configuring for local development
+
+To set up a local copy of PostyBirb for development:
+
+1. clone this repository and `cd` into it.
+2. `npm install` to install the base requirements.
+3. For the `commons`, `ui`, and `electron-app` directories, cd into each and run `npm install && npm run build`. Be sure to start with commons first.
+
+Then, to run the application, enter the `electron-app` directory and run `npm start`. To reload after changes, you will
+need to rebuild all individual modules, like so:
+
+```bash
+cd ../commons && npm run build && cd ../ui && npm run build && cd ../electron-app && npm start
```

Comment:
suggestion: a slightly better one-liner could be this from the base dir
`npm run make && npm run start --prefix electron-app`